pkg/utils/consts.go

```diff
@@ -14,26 +14,31 @@
 package utils
 
 import (
-	"fmt"
 	"os"
+	"strings"
 
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
 var (
-	InitLog            = ctrl.Log.WithName("setup")
-	DashboardNamespace string
-	DataSource         string
+	InitLog           = ctrl.Log.WithName("setup")
+	AllowedNamespaces []string
+	IgnoredNamespaces []string
```

Comment:
I think a blacklist is okay to protect namespaces so `IgnoredNamespaces` is enough. What do you think?

Reply:
> I think a blacklist is okay to protect namespaces so `IgnoredNamespaces` is enough. What do you think?

@yeya24 

If only a few namespaces were allowed to execute chaos, then setting AllowedNamespaces would be a good idea.
If only a few namespaces were not allowed to execute chaos, then setting IgnoredNamespaces would be a good idea.

If both AllowedNamespaces and IgnoredNamespaces are set, only AllowedNamespaces will take effect.